pulsar-client/src/main/java/org/apache/pulsar/client/impl/ClientCnx.java

```diff
@@ -253,7 +253,7 @@ protected ByteBuf newConnectCommand() throws Exception {
     @Override
     public void channelInactive(ChannelHandlerContext ctx) throws Exception {
         super.channelInactive(ctx);
-        log.info("{} Disconnected", ctx.channel());
+        log.warn("{} Disconnected", ctx.channel());
```

Comment:
Disconnection from broker is a normal event in the vast majority of cases. I don't think we should warn here.

Reply:
As described in #12755, if the channel becomes an inactive state may be caused by topic unload or broker reboot.
With a warn log, it's helpful for users to perceive the change and understand the client traffic undulate.

Reply:
My point is that there is (possibly) nothing wrong with the disconnection event and there is no action to be taken here.

Reply:
Agree.